src/app/BE/card/templates/internship/index.js

```diff
@@ -33,9 +34,9 @@ const InternshipCard = ({ parentCode, actions = [], code }) => {
           data={subTitle}
         />
         <HStack>
-          {actions.map(action => (
+          {actions.map((action, index) => (
             <Action
-              key={action}
+              key={`${action}-${index}`}
```

Comment:
Use of index as a key might no be a good alternative, especially if the action are unique in itself. Let's not use index in the keys here if the actions are unique.